src/lib/seam/components/AccessCodeForm/AccessCodeForm.element.ts

```diff
@@ -0,0 +1,12 @@
+import type { ElementProps } from 'lib/element.js'
+import type { AccessCodeFormProps } from 'lib/seam/components/AccessCodeForm/AccessCodeForm.js'
+
+export const name = 'seam-access-code-form'
+
+export const props: ElementProps<AccessCodeFormProps> = {
+  deviceId: 'string',
+  onBack: 'function',
+  className: 'string',
+}
+
+export { AccessCodeForm as Component } from './AccessCodeForm.js'
```

Comment:
@razor-x  added the element here in https://github.com/seamapi/react/pull/271/commits/b0e8c30294ea4d6bdbc1430edd0d589f9389644e

Is there anything else needed for this component?

Reply:
Yea, just need to re-export here: https://github.com/seamapi/react/blob/main/src/lib/seam/components/elements.ts

Reply:
tbh not sure what element does, or where it's referenced, but I've re-exported it here https://github.com/seamapi/react/pull/271/commits/12c54bb21871ec9bb12f2b447d88a0c045d6fbf6

Will remember to do it for any new components!